js/context/UserContext/UserProvider.js

```diff
@@ -30,27 +35,40 @@ class UserProvider extends Component {
     this.getUser();
   };
 
-  updateUser = async (
-    plantName,
-    avatar,
-    streak,
-    savedActivityID,
-    activityRating
-  ) => {
-    await queryUpdateUser(
-      plantName,
-      avatar,
-      streak,
-      savedActivityID,
-      activityRating
-    );
+  updateUser = async (avatar, plantName) => {
+    await queryUpdateUser(avatar, plantName);
     this.getUser();
   };
 
   getUser = () => {
     this.setState({ user: queryGetUser() });
   };
 
+  addActivity = async (activityID, activityRating, dateRecorded) => {
+    await queryAddActivity(activityID, activityRating, dateRecorded);
+    this.getUser();
+  };
+
+  removeActivity = async activityID => {
+    await queryRemoveActivity(activityID);
+    this.getUser();
+  };
+
+  addJournal = async (journalID, imageUrl, hashTags) => {
+    await queryAddJournal(journalID, imageUrl, hashTags);
+    this.getUser();
+  };
+
+  removeJournal = async journalID => {
+    await queryRemoveJournal(journalID);
+    this.getUser();
+  };
+
+  firstTime = async (showOnBoarding, showPrivacyPolicy) => {
```

Comment:
name your function better

Reply:
changed to 2 different functions named toggleShowOnBoarding and toggleShowPrivacyPolicy